Mage.Sets/src/mage/sets/Aetherdrift.java

```diff
@@ -73,6 +73,11 @@ private Aetherdrift() {
         cards.add(new SetCardInfo("Country Roads", 253, Rarity.UNCOMMON, mage.cards.c.CountryRoads.class));
         cards.add(new SetCardInfo("Crash and Burn", 119, Rarity.COMMON, mage.cards.c.CrashAndBurn.class));
         cards.add(new SetCardInfo("Cryptcaller Chariot", 80, Rarity.RARE, mage.cards.c.CryptcallerChariot.class));
+        cards.add(new SetCardInfo("Cursecloth Wrappings", 81, Rarity.RARE, mage.cards.c.CurseclothWrappings.class, NON_FULL_USE_VARIOUS));
+        cards.add(new SetCardInfo("Cursecloth Wrappings", 383, Rarity.RARE, mage.cards.c.CurseclothWrappings.class, NON_FULL_USE_VARIOUS));
+        cards.add(new SetCardInfo("Cursecloth Wrappings", 400, Rarity.RARE, mage.cards.c.CurseclothWrappings.class, NON_FULL_USE_VARIOUS));
+        cards.add(new SetCardInfo("Cursecloth Wrappings", 410, Rarity.RARE, mage.cards.c.CurseclothWrappings.class, NON_FULL_USE_VARIOUS));
```

Comment:
There are build fail on wrong rarity in two cards (400 and 410):
> Error: (rarity) mismatched. MtgJson has Mythic while set file has Rare for DFT - Cursecloth Wrappings - 400

![IMG_1155](https://github.com/user-attachments/assets/141b09d5-c787-4f3e-a501-53ba4f80bef5)